_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,392 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
+
+### Gitter.im for Neovim
+
+Now in addition to the [mailing list][gitter-list] and IRC channel, you can hop
+into the [Neovim chat room][gitter-link] provided through
+[Gitter.im][gitter-link]. It has a convenient web interface but also allows you
+to connect using your favorite IRC client.
```

Comment:
Huh, that's weird. I got a notification that you pushed the change, and it says you did on your profile, but it's not showing up...